server/src/main/java/io/mantisrx/master/jobcluster/job/JobActor.java

```diff
@@ -1380,13 +1395,18 @@ private void initializeRunningWorkers() {
                                     wm.getWorkerIndex(), wm.getWorkerNumber(), Status.TYPE.HEARTBEAT, "",
                                     MantisJobState.Started, System.currentTimeMillis()));
                             worker.processEvent(fakeHB, jobStore);
-                        } catch (Exception e) {
+                        } catch (MissingWorkerPortsException e) {
```

Comment:
Need a closer review here. This is on failover. Idea is to mark bad workers one by one. Then, at the end of this init, resubmit them all together.

Reply:
Also narrowed the exception down to catch just the 3 we need.